owncloudDomain/src/test/java/com/owncloud/android/domain/files/usecases/SaveFileOrFolderUseCaseTest.kt

```diff
@@ -0,0 +1,73 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Christian Schabesberger
+ * Copyright (C) 2020 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.domain.files.usecases
+
+import com.owncloud.android.domain.exceptions.UnauthorizedException
+import com.owncloud.android.domain.files.FileRepository
+import com.owncloud.android.testutil.OC_FILE
+import com.owncloud.android.testutil.OC_FOLDER
+import io.mockk.every
+import io.mockk.spyk
+import io.mockk.verify
+import org.junit.Assert
+import org.junit.Test
+
+class SaveFileOrFolderUseCaseTest {
+    private val fileRepository: FileRepository = spyk()
+    private val useCase = SaveFileOrFolderUseCase(fileRepository)
+    private val file = OC_FILE
+    private val folder = OC_FOLDER
+    private val useCaseParamsFile = SaveFileOrFolderUseCase.Params(file)
+    private val useCaseParamsFolder = SaveFileOrFolderUseCase.Params(folder)
```

Comment:
From my point of view, where is no need to test file and folder separately if there is no difference between them. So you could unify them into a unique test. What do you think?

Reply:
done